icedfrisby-nock.js

```diff
@@ -38,6 +38,14 @@ const factory = superclass =>
       return this
     }
 
+    // Delegates to `intercept()` only if the condition evaluates to `true`.
+    //
+    // @param condition The condition on which the intercept will be set or not
+    // @param setup The setup function, receives `nock` and returns a nock object
+    conditionalIntercept(condition, setup) {
+        return condition === true ? this.intercept(setup) : this
```

Comment:
It seems surprising this would reject truthy values besides `true`. Let's use JavaScript truthiness to this behaves the same as `if`.